open-source-license-acknowledgements-and-third-party-copyrights.md

```diff
@@ -0,0 +1,42 @@
+# Open Source License Acknowledgements and Third-Party Copyrights
+
+Operations-Dashboard is licensed under the terms of the [MIT License](./LICENSE).
```

Comment:
link the License file here

Reply:
the link redirects to this file https://github.com/AfricasVoices/Operations-Dashboard/blob/Commit_libraries_to_project/LICENSE